.gitignore

```diff
@@ -4,3 +4,4 @@ Cargo.lock
 .DS_Store
 
 image-rs/scripts/attestation-agent
+shell.nix
```

Comment:
Is this something left when test?

Reply:
This file is part of my local development setup, so I added to .gitignore.